docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+---
+id: using-gnokey
+---
+
+# Using `gnokey`
+
+## Overview
+In this tutorial, you will learn how to use the `gnokey` binary to interact with
+a Gno.land chain. You will learn how to create state-changing calls, run readonly
+queries without using gas, as well as create, sign, and broadcast airgapped
+transactions for full security.
+
+## Prerequisites
+- **`gno`, `gnokey`, and `gnodev` installed.** Reference the
+  [Local Setup](local-setup/installation.md#2-installing-the-required-tools-) guide for steps
+- **A Gno.land keypair set up.** Reference the
+  [Working with Key Pairs](local-setup/working-with-key-pairs.md) guide for steps
+
+## Interacting with a Gno.land chain
+
+`gnokey` allows you to interact with any Gno.land network, such as the
+[Portal Loop](../concepts/portal-loop.md) testnet.
+
+There are multiple ways anyone can interact with the chain:
+- Transactions - state-changing calls which use gas
+- ABCI queries - read-only calls which do not use gas
+
+Both transactions and ABCI queries can be used via `gnokey`'s subcommands,
+`maketx` and `query`.
+
+## State-changing calls (transactions)
+
+In Gno, there are four types of messages that can change on-chain state:
+- `AddPackage` - adds new code to the chain
+- `Call` - calls a specific path and function on the chain
+- `Send` - sends coins from one address to another
+- `Run` - executes a Gno script against on-chain code
+
+A Gno.land transaction contains two main things:
+- A base configuration where variables such as `gas-fee`, `gas-wanted`, and others
+  are defined
+- A list of messages to execute on the chain
+
+Currently, `gnokey` supports single-message transactions, while multiple-message
+transactions can be created in Go programs, supported by the
+[gnoclient](../reference/gnoclient/gnoclient.md) package.
+
+We will need some testnet GNOTs for each state-changing call. Visit the [Faucet
+Hub](https://faucet.gno.land) to get GNOTs for the currently live Gno testnets.
+
+Let's delve deeper into each of these messages.
+
+### `AddPackage`
```

Comment:
This is a lot of content for one page.  How about creating a parent page that has all the content above and having individual pages for each of the `gnokey` use cases that we want to highlight ?

Reply:
I would propose the following structure.

Have the content in this PR replace the current `gno-tooling/gnokey.md` file by having the intro section be an index page, and each subsection, ie Calls, Queries, Airgapped TXs be its own page. In this case, I would simplify the Working with Key Pairs tutorial in the getting started section to simply allow people to set up a key pair, and add a page under the new `gnokey` section in `gno-tooling` that would cover everything keypair-related. WDYT?

Reply:
Heavy +1 @deelawn